pyproject.toml

```
@@ -16,20 +16,20 @@ license = "MIT"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.8"
-aiohttp = {extras = ["speedups"], version = "^3.7.3"}
-pydantic = "^1.7.3"
+python = "^3.11"
```

```suggestion
python = "^3.10"
```

let's keep it at 3.10 minimum